internal/query/counts.go

```diff
@@ -57,10 +57,6 @@ func (c *Counts) Refresh() {
 		Where("deleted_at IS NULL").
 		Take(c)
 
-	Db().Table("countries").
-		Select("(COUNT(*) - 1) AS countries").
-		Take(c)
```

Comment:
Same as above: Should not be removed / replaced as the country album count may be less. Moments will only be created if enough good photos exist for a certain filter, in this case a country code.